packages/jest-runner/fuzz.ts

```diff
@@ -50,69 +59,131 @@ export const fuzz: FuzzTest = (name, fn) => {
 	// points to the element containing the fuzz function.
 	const testStatePath = currentTestStatePath(testName);
 
+	// The timeout setting is extracted from the test state and defaults to 5s.
+	// Setting timeouts on this level is necessary, as they apply to the
+	// individual inputs and not the whole run.
+	const timeout = currentTimeout();
```

Comment:
Do we want the ability to set the timeout like Jest does?
`it.fuzz("foo", (data) =>{}, 1000);`
It (almost) works for the hooks in the FUZZ-443-jest-hooks PR.
We "just" (!) need to forward it to each `runTestEntry()`.

Reply:
Let's pull this feature out into a dedicated PR.